### PR TITLE
 Update rootfs and installation instructions 

### PR DESCRIPTION
**Pull request by jbruechert**
image

### CODE REVIEW

File: get.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+**Mobile**
+----------
+
+Neon based reference rootfs (recommended)
+----------------------------------------
+
+![](/img/neon.svg){:width="100"}
+
+"Official" image, based on KDE Neon. Targetted at newer devices (devices with 1GB+ RAM).
```
Comment (lnjX) [inline get.md]: not "devices" twice

File: get.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+KDE Neon itself is based upon Ubuntu 16.04 (xenial), (not Ubuntu Touch).
```
Comment (lnjX) [inline get.md]: I don't think the

> (not Ubuntu Touch)

is really necessary

File: get.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+This image is based on the dev-unstable branch of KDE Neon, and always ships the latest version of KDE frameworks, kwin and Plasma Mobile.
```
Comment (lnjX) [inline get.md]: Maybe only

> development branch of KDE Neon

Also versions (plural) for KDE frameworks (plural) is needed

File: get.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+**[<span class="fa fa-angle-right" /> Get it](/neon-arch-reference-rootfs)**
+
+Arch Linux based reference rootfs
+---------------------------------
+
+![](/img/archlinux.png){:width="100"}
+
+Arch Linux based image also made by the Plasma Mobile developers. Known not to be as stable as the Neon based rootfs.
```
Comment (LNJ2) [inline get.md]: I think not should be between be and as stable

File: get.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+PostmarketOS
+------------
+
+![](/img/pmOS.svg){:width="100"}
+
+Distribution especially targetted at older devices, based on Alpine Linux.
+
+Their own description:
+postmarketOS (pmOS), is a touch-optimized, pre-configured Alpine Linux that can be installed on smartphones and other mobile devices. The project is at very early stages of development and is not usable for most users yet.
```
Comment (lnjX) [inline get.md]: Why one time is postmarketOS spelled with a capital letter at the beginning and one time, not? I think a uniform way would be better. Also better remove the "Their own description", as a reader of the website you don't need to know that.

File: get.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+**Desktop**
+-----------
+
+Neon based iso image
+--------------------
+
+![](/img/neon.svg){:width="100"}
+
+Iso image using the same packages as the Neon based reference rootfs, just compiled for amd64.
```
Comment (lnjX) [inline get.md:61]: Article missing for "Iso image"

Also it should be ISO, instead of Iso

~Better replace just with only~

File: img/neon.svg [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+         style="color:#000000;clip-rule:nonzero;display:inline;overflow:visible;visibility:visible;opacity:1;isolation:auto;mix-blend-mode:normal;color-interpolation:sRGB;color-interpolation-filters:linearRGB;solid-color:#000000;solid-opacity:1;fill:#fcfcfc;fill-opacity:1;fill-rule:evenodd;stroke:none;stroke-width:3;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:0.89999998;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;color-rendering:auto;image-rendering:auto;shape-rendering:auto;text-rendering:auto;enable-background:accumulate" />
+    </g>
+  </g>
+</svg>
```
Comment (lnjX) [inline img/neon.svg:164]: Maybe it would be good to optimize this to improve loading times

**Review by lnjX — CHANGES_REQUESTED** [review]

File: get.md (hunk 2 above)
Comment (jbruechert) [reply get.md]: Yeah, I'm not sure there either, but sometimes users where confused and did not understand if Plasma Mobile uses Ubuntu or Ubuntu Touch as base.

File: get.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+Arch Linux based reference rootfs
+---------------------------------
+
+![](/img/archlinux.png){:width="100"}
+
+Arch Linux based image also made by the Plasma Mobile developers. Known not to be as stable as the Neon based rootfs.
+
+**[<span class="fa fa-angle-right" /> Get it](/neon-arch-reference-rootfs)**
+
+PostmarketOS
```
Comment (z3ntu) [inline get.md]: The first p should be small (postmarketOS). On line 40 I guess the capital P is okay, as it's the beginning of a sentence.

File: get.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+**[<span class="fa fa-angle-right" /> Get it](/neon-arch-reference-rootfs)**
+
+postmarketOS
+------------
+
+![](/img/pmOS.svg){:width="100"}
+
+Distribution especially targetted at older devices, based on Alpine Linux.
+
+PostmarketOS (pmOS), is a touch-optimized, pre-configured Alpine Linux that can be installed on smartphones and other mobile devices. The project is at very early stages of development and is not usable for most users yet.
```
Comment (z3ntu) [inline get.md]: or actually make the P there lowercase as the pmOS website writes it lowercase too, sorry 😃

**Pull request merged**